### ISSUE TITLE
Try machine learning crash course

### ISSUE THREAD

**Issue by yuwtennis** [yuwtennis/google-data-engineer#41]
https://developers.google.com/machine-learning/crash-course

**Comment by yuwtennis** [comment]
Completed up to _Reducing Loss/Gradient Decent_
Next _Reducing Loss/Learning Rate_

**Comment by yuwtennis** [comment]
Completed up to _Reducing Loss: Stochastic Gradient DescentNext_ 
Next _Reducing Loss: Stochastic Gradient Descent_

**Comment by yuwtennis** [comment]
Completed up to _Stochastic Gradient Descent_
Doing _First Steps with TF_

**Comment by yuwtennis** [comment]
Completed up to _Generalization_
Next _Training and test sets_

**Comment by yuwtennis** [comment]
Completed up to _Validation Set_
Next _Representation_

**Comment by yuwtennis** [comment]
Completed up to _Representation_
Next _Feature Crosses_
